frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -152,62 +167,59 @@ fn clean<T: Trait>() {
 }
 
 benchmarks! {
-	_ {
-		// User account seed
-		let u in 0 .. 1000 => ();
-	}
+	_ {}
```

Comment:
Is there a way to get rid of this? Seemingly the macro fails without it.

Reply:
No, it is required for the moment. But in general, I am a fan of removing this aspect of the macro all together. I think it actually may cause more problems than it solves in bigger picture.